plugin/forward/README.md

```diff
@@ -6,10 +6,17 @@
 
 ## Description
 
-The *forward* plugin is generally faster (~30+%) than *proxy* as it re-uses already opened sockets
-to the upstreams. It supports UDP, TCP and DNS-over-TLS and uses inband health checking that is
-enabled by default.
-When *all* upstreams are down it assumes healtchecking as a mechanism has failed and will try to
+The *forward* plugin re-uses already opened sockets to the upstreams. It supports UDP, TCP and
+DNS-over-TLS and uses inband health checking.
+
+When it detects an error an health check is performed. This checks runs in a loop, every *0.5s*, for
+a long as the upstream reports unhealthy. Once healthy we stop health checking (until the next
+error). The health checks use a recursive DNS query (`. IN NS`) to get upstream health. Any response
+that is not an network error (REFUSED, NOTIMPL, SERVFAIL, etc) is taken as a healthy upstream. The
+health check uses the same protocol as specific in **TO**. If `max_fails` is set to 0, no checking
+is performed and upstreams will ever be considered unhealthy.
```

Comment:
ever -> never ... "... never be considered unhealthy." ... ?

or perhaps "... always be considered healthy."  (no double negative)